apps/explorer/src/app/components/txs/__generated__/TxsStats.ts

```diff
@@ -0,0 +1,35 @@
+/* tslint:disable */
+/* eslint-disable */
+// @generated
+// This file was automatically generated and should not be edited.
+
+// ====================================================
+// GraphQL query operation: TxsStats
+// ====================================================
+
+export interface TxsStats_statistics {
+  __typename: "Statistics";
+  /**
+   * Average number of orders added per blocks
```

Comment:
```suggestion
   * Average number of orders added per block
```

Reply:
@candida-d These are auto generated from what graphql backend. Nothing we can change on the front-end.

Reply:
@candida-d They are comments btw, not visible to user

Reply:
Ah so they need fixing on that side - good to know! And also good to know they're not user facing